src/reactivity/reactive.ts

```diff
@@ -123,38 +125,136 @@ function observe<T>(obj: T): T {
 
   return observed;
 }
+
+export function shallowReactive<T extends object = any>(obj: T): T {
+  if (__DEV__ && !obj) {
+    warn('"shallowReactive()" is called without provide an "object".');
+    // @ts-ignore
+    return;
+  }
+
+  if (!isPlainObject(obj) || isReactive(obj) || isRaw(obj) || !Object.isExtensible(obj)) {
+    return obj as any;
+  }
+
+  const observed = observe({});
+  // def(obj, ReactiveIdentifierKey, ReactiveIdentifier);
+  markReactive(observe, true);
```

Comment:
shouldn't that be `markReactive(observed, true)` ?

Reply:
yes 😅